docker/Dockerfile

```diff
@@ -51,24 +57,17 @@ RUN	go mod download
 
 COPY	.	.
 
-ARG	BUILD_TAGS
-ENV	BUILD_TAGS=${BUILD_TAGS}
-
 RUN	make livepeer livepeer_cli livepeer_bench livepeer_router
 
-FROM --platform=$TARGETPLATFORM	nvidia/cuda:10.1-cudnn7-runtime AS livepeer-amd64-base
-
-FROM --platform=$TARGETPLATFORM nvidia/cuda:11.6.2-cudnn8-runtime-ubuntu20.04 AS livepeer-arm64-base
-
-FROM livepeer-${TARGETARCH}-base
+FROM --platform=${TARGETPLATFORM}	nvidia/cuda:11.6.2-cudnn8-runtime-ubuntu20.04 AS livepeer-base
 
 ENV	NVIDIA_DRIVER_CAPABILITIES=all
 
 COPY --from=build	/build/	/usr/local/bin/
 COPY --from=build	/usr/bin/grpc_health_probe	/usr/local/bin/grpc_health_probe
 COPY --from=build	/src/tasmodel.pb	/tasmodel.pb
 COPY --from=build	/usr/share/misc/pci.ids	/usr/share/misc/pci.ids
-
-RUN	ldconfig
+# libtensorflow.so is required at runtime, because Ffmpeg DNN filter loads it dynamically
+COPY --from=build	/tf/	/usr/local/
```

Comment:
do we not need a run of `ldconfig`?

Reply:
I don't think we need to run it, when copying libs to system location. It should be run when we modify search paths in `/etc/ld.so.conf`.